crates/swift-bridge-cli/src/clap_exec.rs

```diff
@@ -34,3 +36,30 @@ fn handle_create_package(matches: &ArgMatches) {
 
     create_package(config);
 }
+
+fn parse_resource_pairs(matches: &ArgMatches) -> Vec<(PathBuf, PathBuf)> {
+    let pairs: Vec<String> = matches
+        .get_many("resource")
+        .map(|v| v.cloned().collect())
+        .unwrap_or_default();
+
+    pairs
+        .into_iter()
+        .map(|pair| {
+            let mut split = pair.split(':');
```

Comment:
Can we leave a TODO comment that this breaks for paths that contain `:`, i.e. "/path/to/my\:file.txt".

If we're going to eventually switch to https://github.com/clap-rs/clap/issues/1682 then not worrying about this bug is fine since it won't be a problem when we switch and it's likely that way less than 1% of people would ever run into this.